src/NavbarToggle.js

```diff
@@ -15,6 +18,14 @@ let NavbarToggle = React.createClass({
     $bs_navbar_onToggle: PropTypes.func,
```

Comment:
adding 
```js
$bs_navbar_expanded: PropTypes.bool
```
here will also give you access to the current expanded state of the parent navbar.